bindings/python/tests/test_collection_load.py

```diff
@@ -0,0 +1,46 @@
+# This test assumes that test_collection.py has been run first
+import json
+from pathlib import Path
+
+import dlite
+from dlite.utils import instance_from_dict
+
+
+thisdir = Path(__file__).resolve().parent
+indir = thisdir / "input"
+outdir = thisdir / "output"
+dlite.storage_path.append(thisdir)
+dlite.storage_path.append(thisdir / "entities")
+
+
+# -- Test for PR #750
+
+# check initialisation of collection
+Coll = dlite.get_instance(dlite.COLLECTION_ENTITY)
+assert Coll["dimensions"].size
+
+# Check that we can load collection from dict
+d = json.load(open(indir / "coll.json", "r"))
+coll = instance_from_dict(d)
+assert coll.nrelations == 6
+
+
+# -- Test loading collection
```

Comment:
Uncomment or remove

Reply:
Make two to different storages (with one and many collections) in test_collection.py and load them here.